src/modules/commander/calibration_routines.cpp

```diff
@@ -803,8 +653,8 @@ int calibrate_cancel_subscribe()
 
 	if (vehicle_command_sub >= 0) {
 		// make sure we won't read any old messages
-		struct vehicle_command_s cmd;
-		bool update;
+		vehicle_command_s cmd{};
+		bool update = false;
```

Comment:
```suggestion
		bool updated = false;
```

Reply:
Just noticed it's also leaking the file descriptor.